plugins/golang/skills/fix-cve/SKILL.md

```
@@ -0,0 +1,202 @@
+
+Where `<action>` is:
+- Path A: `update`
+- Path B: `bump go version and update`
+- Path C: `replace`
+
+Use `git commit -s` for sign-off if required.
+
+### Step 6: Optionally create a PR
+
+Format for OpenShift:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)